templates/gatsby-drupal/files/gatsby/README.md

```diff
@@ -0,0 +1,105 @@
+<!-- AUTO-GENERATED-CONTENT:START (STARTER) -->
```

Comment:
Again, the file seems redundant.
Even if kept, the DoP button should be removed as it's for the wrong template.  (Um, actually it's for Pimcore right now... :smile: )